deploy/configmap.yaml

```diff
@@ -21,4 +21,4 @@ data:
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
   Qlogic_qede_QL45000_50G: "1077 1654 1664"
-  
+  Red_Hat_Virtio_network_device: "1af4" "0001" "0000"
```

Comment:
isn't it `1af4 1000 0000` ?

From my logs when I tested it:
```
I0323 11:12:34.454601       1 factory.go:106] device added: [pciAddr: 0000:00:04.0, vendor: 1af4, device: 1000, driver: vfio-pci]
I0323 11:12:34.454613       1 manager.go:139] New resource server is created for dpdk1 ResourcePool
```

Reply:
Ouch, yes! thanks